components/notifier/Dockerfile

```diff
@@ -26,6 +27,8 @@ HEALTHCHECK CMD python -c "from datetime import datetime as dt, timedelta; impor
 COPY --from=compile-image /home/notifier/venv /home/notifier/venv
 COPY notifier/src /home/notifier
 COPY shared_data_model /home/shared_data_model
+# TODO: Move database code from shared_server_code to shared_data_model.
```

Comment:
Please open the ticket in GitHub if you haven't done it and remove the TODO here. The move of the database code involves changes in different places, so the added value of having a TODO here is limited. If you don't want to forget this Dockerfile needs to be changed, it's better to add the reminder in the issue, IMO.

```suggestion
```